src/ontology/cl-edit.owl

```diff
@@ -32258,6 +32260,18 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005385))
 SubClassOf(obo:CL_4042016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011148))
 
+# Class: obo:CL_4047001 (liver-resident natural killer cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26330348") Annotation(oboInOwl:hasDbXref "PMID:26639736") Annotation(oboInOwl:hasDbXref "PMID:27798170") Annotation(oboInOwl:hasDbXref "PMID:32351704") Annotation(oboInOwl:hasDbXref "PMID:34230995") Annotation(oboInOwl:hasDbXref "PMID:35865550") obo:IAO_0000115 obo:CL_4047001 "A natural killer cell resident to the liver, in humans this cell type is distinguished from circulating natural killer cells by CD49a or CD69 gene expression. Liver-resident natural killer cells have also been shown to express CCR5, EOMES, KLRB1, GZMK, and CXCR6 in humans.")
+AnnotationAssertion(terms:contributor obo:CL_4047001 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:contributor obo:CL_4047001 "https://orcid.org/0000-0001-8134-3037")
+AnnotationAssertion(terms:date obo:CL_4047001 "2024-07-29T13:43:11Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26639736") oboInOwl:hasExactSynonym obo:CL_4047001 "liver NK cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35726345") oboInOwl:hasExactSynonym obo:CL_4047001 "lr-NK")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32351704") Annotation(oboInOwl:hasDbXref "PMID:34230995") oboInOwl:hasExactSynonym obo:CL_4047001 "lrNK")
+AnnotationAssertion(rdfs:label obo:CL_4047001 "liver-resident natural killer cell")
+EquivalentClasses(obo:CL_4047001 ObjectIntersectionOf(obo:CL_0000623 ObjectSomeValuesFrom(obo:RO_0001025 obo:UBERON_0002107)))
+
```

Comment:
these might be other synonyms for liver resident natural killer cells:

- hepatic natural killer cells
- hepatic NK
- he-NK
PMID: [31114585](https://pubmed.ncbi.nlm.nih.gov/31114585)

- intrahepatic NK
PMID: [33828559](https://pubmed.ncbi.nlm.nih.gov/33828559)

Reply:
Thanks Alida, added!